prbt_hardware_support/include/prbt_hardware_support/pilz_modbus_client_exception.h

```diff
@@ -23,16 +23,16 @@
 namespace prbt_hardware_support
 {
   /**
-   * @brief Expection thrown by prbt_hardware_support::PilzModbusReadClient
+   * @brief Expection thrown by prbt_hardware_support::PilzModbusClient
    */
-  class PilzModbusReadClientException : public std::runtime_error
+  class PilzModbusClientException : public std::runtime_error
   {
     public:
-      PilzModbusReadClientException( const std::string& what_arg ):
+      PilzModbusClientException( const std::string& what_arg ):
         std::runtime_error(what_arg)
       {
       }
   };
 }
 
-#endif // PRBT_HARDWARE_SUPPORT_READ_CLIENT_EXCEPTION_H
\ No newline at end of file
+#endif // PRBT_HARDWARE_SUPPORT_READ_CLIENT_EXCEPTION_H
```

Comment:
change the precompiler variable PRBT_HARDWARE_SUPPORT_READ_CLIENT_EXCEPTION_H -> PRBT_HARDWARE_SUPPORT_MODBUS_CLIENT_EXCEPTION_H